spec/unit/pdk/util/template_uri_spec.rb

```diff
@@ -161,7 +161,7 @@
 
           it 'returns the specified template' do
             allow(Gem).to receive(:win_platform?).and_return(true)
-            expect(template_uri.to_s).to eq("C:\\cli-templates##{described_class.default_template_ref}")
+            expect(template_uri.to_s).to eq('C:\cli-templates#master')
```

Comment:
nit: Although it's not required, I've found just always using double slash makes it far less confusing e.g. `\'` and `\\` is still are valid escape sequences in single quoted strings.